chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -743,6 +742,20 @@ def slaves
       ports.map{ |i| ::Nic.new(i) }
     end
 
+    def add_slave(slave)
+      slave = self.class.coerce(slave)
+      unless ::Nic.exists?(slave)
+        raise ::ArgumentError.new("#{slave} does not exist, cannot add to bridge#{@nic}")
+      end
+      return if self.slaves.member?(slave)
+      if current_master = slave.master()
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.